python/ray/experimental/channel/intra_process_channel.py

```diff
@@ -0,0 +1,59 @@
+import uuid
+from typing import Any
+
+import ray
+from ray.experimental.channel import ChannelContext
+from ray.experimental.channel.common import ChannelInterface
+from ray.util.annotations import PublicAPI
+
+
+@PublicAPI(stability="alpha")
+class IntraProcessChannel(ChannelInterface):
+    """
+    IntraProcessChannel is a channel for communication between two tasks in the same
+    worker process. It writes data directly to the worker's serialization context
```

Comment:
Be more specific that this is `_SerializationContext` or the serialization context of `ChannelContext`? There is another serialization context in `serialization.py`, so it may be a bit ambiguous

Reply:
Updated https://github.com/ray-project/ray/pull/45591/commits/777a2847d8b64add7b1d495bba72c4ed7ddb2f23